src/jenesis/cmd/attach.py

```diff
@@ -32,25 +32,22 @@ def run(args: argparse.Namespace):
     if args.contract not in selected_profile.deployments:
         print('Deployment not found in project')
         return 1
-    deployment = selected_profile.deployments[args.contract]
 
     client = LedgerClient(selected_profile.network)
-
-    project_contracts = {contract.name: contract for contract in detect_contracts(project_path)}
+    contract_to_attach = parse_contract(project_path, args.contract)
```

Comment:
Should this be `args.contract` or `deployment.contract`? I think that `args.contract` is actually the name of a deployment (which is a bit confusing).